modules/ROOT/pages/usage-metrics.adoc

```diff
@@ -0,0 +1,142 @@
+= Usage and Pricing Metrics Reference
+:page-aliases: pricing-metrics.adoc
+
+Anypoint Platform captures usage metrics across products, but not all metrics are billable or reflected in usage reports.
+
+[cols=3*]
+|===
+|Product |Metric Description |Usage Details
+
+.3+|Mule Runtime	
+|_Mule flow_: A flow within a deployed and running Mule app that contains a Mule event source or route APIKit request. 
```

Comment:
Why italics here?

Reply:
Can you just say "running"? Do you need "deployed"?

Reply:
It's that apps can be deployed but might not be running just because they are deployed.